R/stats-mlm-tidiers.R

```diff
@@ -0,0 +1,88 @@
+#' @templateVar class lm
+#' @template title_desc_tidy
+#'
+#' @param x An `lm` object created by [stats::lm()].
+#' @template param_confint
+#' @template param_quick
+#' @template param_unused_dots
+#'
+#' @evalRd return_tidy(regression = TRUE)
+#'
+#' @details If the linear model is an `mlm` object (multiple linear model),
+#'   there is an additional column `response`.
+#'
+#'   If you have missing values in your model data, you may need to refit
+#'   the model with `na.action = na.exclude`.
+#'
+#' @examples
+#'
+#' mod <- lm(cbind(mpg, disp) ~ wt, mtcars)
+#' tidy(mod, conf.int = TRUE)
+#' @aliases lm_tidiers
+#' @export
+#' @seealso [tidy()]
+#' @family lm tidiers
+#' @rdname tidy.mlm
+#' @export
+
+tidy.mlm <- function(x,
+                     conf.int = FALSE,
+                     conf.level = .95,
+                     quick = FALSE,
+                     ...) {
+  # if only term and estimate columns are needed
+  if (quick) {
```

Comment:
Let's get rid of the `quick` logic, I'm going to strip all of it out for 0.7.0. It's hard to maintain and I don't think anyone uses it.

Reply:
Awesome! Happy to get rid of this. Didn't realize we were deprecating this in `0.7.0`.